frameworks/g-cloud-9/questions/declaration/fraudAndTheft.yml

```diff
@@ -0,0 +1,14 @@
+question: >
+  Please confirm whether, within the past 5 years, anyone who represents, supervises or has control in your organisation or a partner or parent organisation has been convicted of any offence that relates to fraud affecting the European Communities’ financial interests as defined by article 1 of the Convention on the Protection of the Financial Interests of the European Communities, including:
+
+    - the common law offence of cheating the Revenue (HMRC)
```

Comment:
The declaration content here is copied direct from DOS2 and is due for a complete review, so I'm tempted to leave this as it is for now.